### PR TITLE
Utilities for Plutarch scripts

### LINKED ISSUE #76: Utilities for Plutarch scripts
Implement functions in `Plutus.Model.Validator.Vx` for Plutarch.

- From the appropriate Plutarch type to `TypedValidator`
- For validators, minting policies and staking validators
- For V1 and V2

Remember to add a CHANGELOG entry and bump the minor version number (`C` in `A.B.C.D`).

### PR DESCRIPTION
**Pull request by Geometer1729**
Closes #76

### CODE REVIEW

**Comment by Geometer1729** [comment]
This is failing CI but not in any ways the main branch isn't. 
We should probably make an issue about the haddock check failing or remove the check if it doesn't matter.

**Comment by AriFordsham** [comment]
CI is broken for haddock for some reason. I haven't had a chance to fix/issue/disable.

File: CHANGELOG [hunk 1]
```diff
@@ -5,6 +5,16 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to the [Haskell Package Versioning Policy](https://pvp.haskell.org/).
 
+## [0.5.2.0] - 2023-01-02
+
+### Added
```
Comment (AriFordsham) [inline CHANGELOG:10]: Maybe a note here about what these do?

**Review by AriFordsham — CHANGES_REQUESTED** [review]
Looks excellent!

**Review by AriFordsham — APPROVED** [review]
LGTM!

**Pull request merged**